common/src/main/java/com/microsoft/identity/common/internal/fido/CredManFidoManager.kt

```diff
@@ -23,28 +23,49 @@
 package com.microsoft.identity.common.internal.fido
 
 import android.content.Context
+import android.os.Build
 import androidx.credentials.CredentialManager
 import androidx.credentials.GetCredentialRequest
 import androidx.credentials.GetPublicKeyCredentialOption
 import androidx.credentials.PublicKeyCredential
+import androidx.credentials.exceptions.NoCredentialException
+import com.microsoft.identity.common.internal.providers.oauth2.WebViewAuthorizationFragment
+import com.microsoft.identity.common.java.flighting.CommonFlight
+import com.microsoft.identity.common.java.flighting.CommonFlightsManager
+import com.microsoft.identity.common.java.opentelemetry.AttributeName
+import com.microsoft.identity.common.logging.Logger
+import io.opentelemetry.api.trace.Span
 
 /**
  * Makes calls to the Android Credential Manager API in order to return an attestation.
  */
-class CredManFidoManager (val context: Context) : IFidoManager {
+class CredManFidoManager (val context: Context,
+                          val fragment: WebViewAuthorizationFragment) : IFidoManager {
+
+    val TAG = CredManFidoManager::class.simpleName.toString()
```

Comment:
> val TAG = CredManFidoManager::class.simpleName.toString() [](http://example.com/codeflow?start=3&length=58)

TAG in companion object

Reply:
37b32956aa6aa6df1f728faa590b0f9672a908a8